File: Java/BookStoreOOP/src/model/Reader.java
package model;

public class Reader {
    private String id;
    private String name;
    private Boolean gender;
    private ReaderCategory category;

    public Reader(String id, String name, Boolean gender, ReaderCategory category) {
        this.id = id;
        this.name = name;
        this.gender = gender;
        this.category = category;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public Boolean getGender() {
        return gender;
    }

    public void setGender(Boolean gender) {
        this.gender = gender;
    }

    public ReaderCategory getCategory() {
        return category;
    }

    public void setCategory(ReaderCategory category) {
        this.category = category;
    }
}

File: C#/BookStoreOOP/BookStoreOOP/Model/Reader.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace BookStoreOOP.Model
{
    public class Reader
    {
        private string id;
        private string name;
        private bool gender;
        private ReaderCategory category;

        public Reader(string id, string name, bool gender, ReaderCategory category)
        {
            this.id = id;
            this.name = name;
            this.gender = gender;
            this.category = category;
        }

        public string getId()
        {
            return id;
        }

        public void setId(string id)
        {
            this.id = id;
        }

        public string getName()
        {
            return name;
        }

        public void setName(string name)
        {
            this.name = name;
        }

        public bool getGender()
        {
            return gender;
        }

        public void setGender(bool gender)
        {
            this.gender = gender;
        }

        public ReaderCategory getCategory()
        {
            return category;
        }

        public void setCategory(ReaderCategory category)
        {
            this.category = category;
        }
    }
}
